changelog/6629.bugfix.md

```diff
@@ -0,0 +1 @@
+Fixed a bug when setting multiple Sanic workers. Previously, if the number was set higher than 1 and you were using a custom lock store, it would reject because of a strict check to use a Redis lock store.
```

Comment:
```suggestion
Fixed a bug that occurred when setting multiple Sanic workers in combination with 
a custom [Lock Store](lock-stores.mdx). Previously, if the number was set 
higher than 1 and you were using a custom lock store, it would reject because of a 
strict check to use a [Redis Lock Store](lock-stores.mdx#RedisLockStore).
```

Reply:
Fixed!